frontends/web/src/i18n/config.js

```diff
@@ -34,7 +34,11 @@ export default {
                 return;
             }
             apiGet('native-locale').then(locale => {
-                if (typeof locale === 'string' && locale) {
+                if (
+                    typeof locale === 'string'
+                    && locale
+                    && !['C.UTF-8', 'C'].includes(locale.toUpperCase())
```

Comment:
How about we test whether i18next supports this locale, regardless whether it's C, and fall back to `defaultUserLanguage`? Just thinking future-proof, in case a user has weird LANG set.

If there's no function in i18next then ok, I guess.

Reply:
@x1ddos PTAL